documents-webapp/src/main/webapp/vue-app/documents/components/DocumentsMain.vue

```diff
@@ -5,20 +5,26 @@
     role="main"
     flat>
     <div class="pa-3 white">
-      <documents-header
-        class="py-2"
-        :is-mobile="isMobile" />
-      <documents-body
-        :view-extension="selectedViewExtension"
-        :files="files"
-        :groups-sizes="groupsSizes"
-        :page-size="pageSize"
-        :offset="offset"
-        :limit="limit"
-        :has-more="hasMore"
-        :sort-field="sortField"
-        :ascending="ascending"
-        :loading="loading" />
+      <div v-if="!filesLoad">
+        <documents-header-left
+          class="py-2" />
```

Comment:
Why the isMobile prop is removed ?

Reply:
it is a mistake when i rebase with branch feature/document, i will add it